src/components/FeatureViews/AdminPortal/UnpaidSessionCard.js

```diff
@@ -61,7 +62,7 @@ const useStyles = makeStyles((theme) => ({
 const UnpaidSessionCard = ({unpaidStudent}) => {
     const classes = useStyles();
     
-    const studentName = `${unpaidStudent.student.user.firstName} ${unpaidStudent.student.user.lastName}`
+    const studentName = fullName(unpaidStudent.student.user.firstName, unpaidStudent.student.user.lastName)
```

Comment:
Be sure to test these things... the `fullName` function takes a user object and returns `user.firstName + user.lastName` This will break the function